siemens_cp1616/launch/test_io_device.launch

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<launch>
+  <node 
+    name="siemens_cp1616_io_device_wrapper"
+    pkg="siemens_cp1616"
+    type="siemens_cp1616_io_device_wrapper"
+    output="screen"/>
+  <rosparam>
+    filepath: "/home/durovsky/catkin_ws/src/siemens_cp1616/siemens_cp1616/config/test_io_device_config.yaml"
```

Comment:
changed to 
 `<param name="filepath" value="$(find siemens_cp1616)/config/test_io_device_config.yaml"/>`